### PR TITLE
ISSUE #2259: Update authorizing official to senior official

### PR DESCRIPTION
**Pull request by therealslimhsiehdy**
## Ticket

Resolves [#2259](https://github.com/cisagov/manage.get.gov/issues/2259)

## Changes

- [before-you-request-steps.md changes](https://federalist-877ab29f-16f6-4f12-961c-96cf064cf070.sites.pages.cloud.gov/preview/cisagov/get.gov/rh/2259-update-ao-to-so/domains/before/)
Screenshot 2024-06-20 at 12 00 15 PM

<br>
<br>

- [information_needed_for_domain_request.md](https://federalist-877ab29f-16f6-4f12-961c-96cf064cf070.sites.pages.cloud.gov/preview/cisagov/get.gov/rh/2259-update-ao-to-so/domains/before/#information-you-will-need-to-complete-the-domain-request)
Screenshot 2024-06-20 at 12 08 42 PM

<br>
<br>

- [domains_before.md](https://federalist-877ab29f-16f6-4f12-961c-96cf064cf070.sites.pages.cloud.gov/preview/cisagov/get.gov/rh/2259-update-ao-to-so/domains/before/#your-senior-official)
Screenshot 2024-06-20 at 12 09 22 PM

<br>
<br>

- [domains_election.md](https://federalist-877ab29f-16f6-4f12-961c-96cf064cf070.sites.pages.cloud.gov/preview/cisagov/get.gov/rh/2259-update-ao-to-so/domains/election-offices/#the-gov-basics-for-election-offices)
Screenshot 2024-06-20 at 12 11 39 PM

<br>
<br>

- [domains_eligibility.md](https://federalist-877ab29f-16f6-4f12-961c-96cf064cf070.sites.pages.cloud.gov/preview/cisagov/get.gov/rh/2259-update-ao-to-so/domains/eligibility/)
Screenshot 2024-06-20 at 12 12 24 PM

<br>
<br>

- [domains_requirements.md](https://federalist-877ab29f-16f6-4f12-961c-96cf064cf070.sites.pages.cloud.gov/preview/cisagov/get.gov/rh/2259-update-ao-to-so/domains/requirements/)
Screenshot 2024-06-20 at 12 13 15 PM

<br>
<br>

- [help_domain-management.md](https://federalist-877ab29f-16f6-4f12-961c-96cf064cf070.sites.pages.cloud.gov/preview/cisagov/get.gov/rh/2259-update-ao-to-so/help/domain-management/)
Screenshot 2024-06-20 at 12 14 31 PM
Screenshot 2024-06-20 at 12 14 52 PM

#### Satisfied acceptance criteria and met development standards

- [ ] Met the acceptance criteria, or will meet them in a subsequent PR
- [ ] Created/modified automated tests
- [ ] Added at least 2 developers as PR reviewers (only 1 will need to approve)
- [ ] Messaged on Slack or in standup to notify the team that a PR is ready for review
- [ ] Changes to “how we do things” are documented in READMEs and or onboarding guide
- [ ] If any model was updated to modify/add/delete columns, makemigrations was ran and the assoicated migrations file has been commited.

#### Ensured code standards are met (Original Developer)

- [ ] All new functions and methods are commented using plain language
- [ ] Did dependency updates in Pipfile also get changed in requirements.txt?
- [ ] Interactions with external systems are wrapped in try/except
- [ ] Error handling exists for unusual or missing values

#### Validated user-facing changes (if applicable)

- [ ] New pages have been added to .pa11yci file so that they will be tested with our automated accessibility testing
- [ ] Checked keyboard navigability
- [ ] Tested general usability, landmarks, page header structure, and links with a screen reader (such as Voiceover or ANDI)
- [ ] Add at least 1 designer as PR reviewer

#### Reviewed, tested, and left feedback about the changes

- [ ] Pulled this branch locally and tested it
- [ ] Reviewed this code and left comments
- [ ] Checked that all code is adequately covered by tests
- [ ] Made it clear which comments need to be addressed before this work is merged
- [ ] If any model was updated to modify/add/delete columns, makemigrations was ran and the assoicated migrations file has been commited.

#### Ensured code standards are met (Code reviewer)

- [ ] All new functions and methods are commented using plain language
- [ ] Interactions with external systems are wrapped in try/except
- [ ] Error handling exists for unusual or missing values
- [ ] (Rarely needed) Did dependency updates in Pipfile also get changed in requirements.txt?

#### Validated user-facing changes as a developer

- [ ] New pages have been added to .pa11yci file so that they will be tested with our automated accessibility testing
- [ ] Checked keyboard navigability
- [ ] Meets all designs and user flows provided by design/product
- [ ] Tested general usability, landmarks, page header structure, and links with a screen reader (such as Voiceover or ANDI)

- [ ] Tested with multiple browsers, the suggestion is to use ones that the developer didn't (check off which ones were used)
  - [ ] Chrome
  - [ ] Microsoft Edge
  - [ ] FireFox
  - [ ] Safari

- [ ] (Rarely needed) Tested as both an analyst and applicant user

**Note:** Multiple code reviewers can share the checklists above, a second reviewers should not make a duplicate checklist

#### Verified that the changes match the design intention

- [ ] Checked that the design translated visually
- [ ] Checked behavior
- [ ] Checked different states (empty, one, some, error)
- [ ] Checked for landmarks, page heading structure, and links
- [ ] Tried to break the intended flow

#### Validated user-facing changes as a designer

- [ ] Checked keyboard navigability
- [ ] Tested general usability, landmarks, page header structure, and links with a screen reader (such as Voiceover or ANDI)

- [ ] Tested with multiple browsers (check off which ones were used)
  - [ ] Chrome
  - [ ] Microsoft Edge
  - [ ] FireFox
  - [ ] Safari

- [ ] (Rarely needed) Tested as both an analyst and applicant user

### CODE REVIEW

File: _includes/content-blocks/before-you-request-steps.md
Comment (erinysong) [inline _includes/content-blocks/before-you-request-steps.md:5]: small nit: `...-a-senior-official-...`

File: _includes/content-blocks/before-you-request-steps.md
Comment (erinysong) [inline _includes/content-blocks/before-you-request-steps.md:6]: Good catch!

**Review by erinysong — APPROVED** [review]
Verified that all content changes display on the cloud pages sandbox. Requested a small spelling change but LGTM to merge after that is fixed!

**Review by kristinacyin — APPROVED** [review]
Looks good! ⭐

File: pages/domains/domains_election.md
Comment (Katherine-Osos) [inline pages/domains/domains_election.md:33]:
```suggestion
All domain requests must be approved by a senior official. This person must be in a role of significant, executive responsibility within the organization. 
```

File: pages/domains/domains_eligibility.md
Comment (Katherine-Osos) [inline pages/domains/domains_eligibility.md:30]: This should be "**a** senior official" - not updating because I think that will break links in various places. Can you update here and in the links?

File: pages/domains/domains_election.md
Comment (therealslimhsiehdy) [reply pages/domains/domains_election.md]: Updated! Screenshot 2024-06-26 at 4 38 58 PM

File: pages/domains/domains_eligibility.md
Comment (therealslimhsiehdy) [reply pages/domains/domains_eligibility.md]: Here are the new links to click through!

[Before you request](https://federalist-877ab29f-16f6-4f12-961c-96cf064cf070.sites.pages.cloud.gov/preview/cisagov/get.gov/rh/2259-update-ao-to-so/domains/before/) (this one has two links on this page for the senior official portion which should lead to the link below!)
[The main link for approval from a senior official](https://federalist-877ab29f-16f6-4f12-961c-96cf064cf070.sites.pages.cloud.gov/preview/cisagov/get.gov/rh/2259-update-ao-to-so/domains/eligibility/#you-must-have-approval-from-a-senior-official-within-your-organization)

**Review by Katherine-Osos — APPROVED** [review]

**Pull request merged**
